wicket-core/src/main/java/org/apache/wicket/markup/html/list/ListView.java

```diff
@@ -652,4 +653,24 @@ public final void setModelObject(List<T> object)
 	{
 		setDefaultModelObject(object);
 	}
+
+	/**
+	 * Returns a new ListVIew
+	 * @param id component id
+	 * @param model model containing a list of items
+	 * @param consumer lambda to use for populating the ListView
+	 * @param <T> type of the items in the model
+	 * @return a new ListView
+	 */
+	public static <T> ListView<T> populateItem(final String id, final IModel<? extends List<T>> model, final SerializableConsumer<ListItem<T>> consumer) {
```

Comment:
Instead of populateItem(...) how about listView(...) -- make the static mirror the class name?

The Lambdas class is interesting!  I had missed that before...